src/scss/_responsive-flat.scss

```diff
@@ -0,0 +1,63 @@
+////
+/// @group o-table
+/// @link http://registry.origami.ft.com/components/o-table
+////
+
+/// Make a table 'ResponsiveFlat'
+@mixin oTableResponsiveFlat {
+	display: block;
```

Comment:
`display: block` shouldn't be needed here because it's added by the following float :)